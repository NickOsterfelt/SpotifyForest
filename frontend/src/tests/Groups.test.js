import React from "react";
import { render } from "@testing-library/react";
import Groups from "../routes/Groups";

it("renders without crashing", function() {
  render(<Groups />);
});

it("matches snapshot with no Groups and user", function() {
  const { asFragment } = render(<Groups />);
  expect(asFragment()).toMatchSnapshot();
});
