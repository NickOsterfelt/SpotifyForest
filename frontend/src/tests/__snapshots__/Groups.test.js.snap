// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot with no Groups and user 1`] = `
<DocumentFragment>
  <div
    class="container mt-2 "
  >
    <div
      class="row d-flex justify-content-center"
    >
      <div
        class="col-8 bg-secondary rounded text-light p-3 px-5 mt-3 text-left"
      >
        <div
          class=""
        >
          <h4>
            Looks like you are not in any groups yet!
          </h4>
          <h5>
            Click the "Find Groups" button to find groups that have similar music tastes.
          </h5>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
